### DISCUSSION TITLE
Add the ability to set a skin specifically for use when editing beatmaps

### DISCUSSION THREAD

**Discussion by 232313** [ppy/osu#13114] [Ideas]
In osu!Stable, we have seen this feature presented, but only for the default skin, where you can select whether or not the default skin or your current skin will be used when in the editor. 

What I'm thinking of is adding this same feature, but with the ability to choose which skin to use in the editor, so, say, you can have a mapping skin and a different playing skin.

**Comment by bdach** [comment]
#7193 is related.
